src/command/subgraph/push.rs

```diff
@@ -10,9 +11,10 @@ use crate::command::RoverStdout;
 #[derive(Debug, Serialize, StructOpt)]
 pub struct Push {
     /// Path of .graphql/.gql schema file to push
-    #[structopt(name = "SCHEMA_PATH", parse(from_os_str))]
+    /// Can pass `-` to use stdin instead of a file
+    #[structopt(long, short = "s")]
```

Comment:
we should likely have a parser function for this that does the check for `-` or if it's a valid path. the docs for custom parsers like that are [here](https://docs.rs/structopt/0.3.21/structopt/#custom-string-parsers)

Reply:
Ooh, yeah! I like that idea. I was working on building one for the graph id work, but that'd work great here too!

Do you think it'd make more sense to have that parser return the schema document itself (which will be used in at least 4 commands) or some struct with either a PathBuf or String?

Reply:
i'd say the parser should take in the raw argument (`-` or a path string) and then convert it to a schema document

Reply:
🆗 🆗